commons/ava_services/gecko_service_initializer_core.go

```diff
@@ -111,8 +112,8 @@ func (g GeckoServiceInitializerCore) InitializeMountedFiles(osFiles map[string]*
 		inside the controller, therefore it is not in Docker, therefore it does not have network access to the bootstrapped node.
 	 */
 
-	certFilePointer := osFiles[stakingTlsCertPath]
-	keyFilePointer := osFiles[stakingTlsKeyPath]
+	certFilePointer := osFiles[stakingTlsCertFileId]
```

Comment:
why change these from paths to IDs? paths feel more intuitive, but i imagine there's a good reason for this

Reply:
Yep, so actually they were never actually "paths" to begin with; that was a misnomer. What they really are is keys to the map of files that's passed in between the `GetRequestedFiles`/`InitializeRequestedFiles` functions, since the actual filepath of where those files live will be an implementation detail handled by the Kurtosis code (since they'll need to live inside the test volume, which is mostly handled by Kurtosis code).